pkg/clusteragent/autoscaling/workload/loadstore/workload_status.go

```diff
@@ -0,0 +1,107 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package loadstore
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	apiServerCommon "github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver/common"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// LocalAutoscalingWorkloadCheckResponse is the response type of the autoscaling workload check.
+type LocalAutoscalingWorkloadCheckResponse struct {
```

Comment:
nit: I think naming here can be improved to make it clearer that the output is metric entities
maybe something along the lines of `LoadstoreMetricInfo`/`LocalMetricDump`?